src/commands/trust/sign.ts

```diff
@@ -34,10 +35,33 @@ export default class Sign extends SfdxCommand {
       char: 't',
       required: false,
       description: messages.getMessage('target'),
+      exclusive: ['tarpath'],
+    }),
+    tarpath: flags.string({
+      description: messages.getMessage('tarPath'),
+      exclusive: ['target'],
     }),
   };
 
   public async run(): Promise<SigningResponse> {
-    return packAndSignApi.doPackAndSign(this.flags as SigningOpts, this.ux);
+    packAndSignApi.setUx(this.ux);
+
+    if (this.flags.tarpath) {
```

Comment:
Not necessary by any means, but it might be nice to encapsulate this in a method in the packAndSignApi file, e.g.

```typescript
if (this.flags.tarpath) {
  return packAngSignApi.doSign(this.flags.tarPath)
} else {
  return packAndSignApi.doPackAndSign(this.flags)
}
```